frontend/src/components/layout/navigation/Navigation.module.scss

```diff
@@ -24,7 +24,6 @@
   color: inherit;
 
   &.is-active {
-    padding-top: $spacing-xs;
```

Comment:
The currently-active link was slightly off-centre due to this padding, which was more noticeable now that the strings are longer.